src/lib/fetchData.ts

```diff
@@ -34,7 +35,14 @@ export async function fetchDeathsData() {
         const pageData = await fetchPage(page);
         allRows.push(...pageData);
     }
+    
+    const result = allRows.map((death){
+        death.modified_name = toKebabCase(death.name_en);
```

Comment:
Code breaks here. Missing arrow for function declaration 
```suggestion
    
    const result = allRows.map((death) => {
        death.modified_name = toKebabCase(death.name_en);
```